clang/include/clang/AST/PrettyPrinter.h

```diff
@@ -159,6 +159,17 @@ struct PrintingPolicy {
   /// When true, suppress printing of lifetime qualifier in ARC.
   unsigned SuppressLifetimeQualifiers : 1;
 
+  /// When true prints a canonical type instead of an alias. E.g.
```

Comment:
>  /// When true prints a canonical type instead of an alias. E.g.

Why is it called `SuppressTypedefs` and not `SuppressTypeAliases`?

Reply:
It works for a Clang Type which have `TypeClass` equal `Type::Typedef`.
In cases of `typedef` and `using `for creation type alias, in both cases a type will be have `Type::Typedef` type class.
You didn't like what I called it as alias here?

Reply:
I think there is a mix of C++ specifier for adding type aliases and Clang's types.
I don't think that `Typedef` is a good name for Clang's type as it covers more than just `typedef` specifier. This looks like some legacy code asking for refactoring.

If current implementation is aligned with the existing Clang design, I'm okay to keep this naming. Refactoring should be proposed to LLORG first.